ui/src/app/utils/access-utils.tsx

```diff
@@ -182,6 +182,12 @@ export const getAccessModuleConfig = (
         refreshAction: async () => await profileApi().syncTwoFactorAuthStatus(),
       }),
     ],
+    [
+      AccessModule.RASLINKIDME,
```

Comment:
The data access requirements page would not load correctly if we do not have something corresponding to our new access module `RASLINKIDME`. This is a placeholder, and should be replaced in https://precisionmedicineinitiative.atlassian.net/browse/RW-10366

Reply:
This is a good default, thanks